.gitignore

```diff
@@ -183,4 +183,5 @@ ex.c
 **/libtorch*
 *.pem
 *.key.pem
-*.html
\ No newline at end of file
+*.html
+*~
```

Comment:
What is this for ?

Reply:
That's because emacs creates backup files when creating or editing a file that terminates with <filename>~

Reply:
Ok sounds good